### PR TITLE
FIX(installer): Re-add URL protocol registration registry keys on install

### LINKED ISSUE #5473: Internet links with protocoltype mumble:// not working
### Description

After installation of Mumble on Windows the Internetshortcuts with mumble://my.url.to are not working

### Steps to reproduce

1. create a new shortcut with die destination to your mumbleserver for example mumble://my.url.to 
2. windows cant resolve mumble:// /protocoll "mumble" ist not linked to any Program"

### Mumble version

_No response_

### Mumble component

Client

### OS

Windows

### Reproducible?

Yes

### Additional information

_No response_

### Relevant log output

_No response_
mumble

### PR DESCRIPTION
**Pull request by magneticflux-**
In the installer refactor (88e17868493b7e08fe3339069c11cbc5c39e62e5) these registry keys were removed, so new installs did not register a URL protocol and users weren't able to use `mumble://` links.
This is a direct translation of `Files.wxs` to equivalent WixSharp code.

Closes #5473

### Checks

- [x] My commits follow the [commit guidelines](https://github.com/mumble-voip/mumble/blob/master/COMMIT_GUIDELINES.md)

### CODE REVIEW

**Comment by Krzmbrzl** [comment]
Did you verify yet that this does indeed make the mumble-URLS work again?

**Comment by magneticflux-** [comment]
Well I borked my local build just as I thought I fixed it, so I'm trying to download the PR artifact, but that's not working now either.

**Comment by magneticflux-** [comment]
Looks like the [File Table](https://docs.microsoft.com/en-us/windows/win32/msi/file-table) isn't being populated the same, so `[#mumble.exe]` doesn't work anymore.

**Comment by magneticflux-** [comment]
Okay, it builds, installs, and the registry keys work now!

**Comment by Krzmbrzl** [comment]
Great! :+1:

**Comment by magneticflux-** [comment]
Should I also make a backport PR to 1.4.x, or does that get done internally?

**Comment by Krzmbrzl** [comment]
I will do that once @ZeroAbility has reviewed this PR. I have an automation tool for that, so it's done very quickly. But thanks for the offer!

**Review by ZeroAbility — COMMENTED** [review]
>This is a direct translation of `Files.wxs` to equivalent WixSharp code.

I would clarify this comment. It sounds like you added everything in the old WiX project instead of just adding the registry entries.

If the resulting installer works and the desired behavior occurs when clicking a URL, then I see no reason not to approve it.

**Review by ZeroAbility — APPROVED** [review]
I approve this change. A second pair of eyes would be good before merging.

**Comment by davidebeatrici** [comment]
I assume the registry keys created by the installer are automatically removed on uninstall, right?

**Comment by ZeroAbility** [comment]
It might be worth it to review these:

https://github.com/oleg-shilo/wixsharp/blob/d626a203fddb8b66ad53c6ae604607068fb432b6/Source/src/WixSharp/RegValue.cs#L299-L307

**Comment by magneticflux-** [comment]
The installer should already track and remove the registry keys it creates (according to the Wix site here: https://wixtoolset.org/documentation/manual/v3/xsd/wix/registrykey.html).

I think the "force" option is only needed if we want to remove something that wasn't put there by this installer.

**Comment by ZeroAbility** [comment]
@magneticflux - Sometimes WiX is not as advertised, and they are not always very responsive when it comes to bugs. If you are able to just install Mumble fresh without the registry entry and do an uninstall to test removal of the registry entry, I think we would all consider that a sign off.

**Comment by magneticflux-** [comment]
@ZeroAbility Just confirmed that a fresh install and uninstall removes the registry keys.

**Review by Krzmbrzl — APPROVED** [review]

**Comment by Krzmbrzl** [comment]
Thanks for fixing this :+1:

**Pull request merged**
